vql/sigma/evaluator/modifiers/modifiers.go

```diff
@@ -26,6 +26,7 @@ func getComparator(comparators map[string]Comparator, modifiers ...string) (Comp
 	// A valid sequence of modifiers is ([ValueModifier]*)[Comparator]?
 	// If a comparator is specified, it must be in the last position and cannot be succeeded by any other modifiers
 	// If no comparator is specified, the default comparator is used
+	// TODO: The explanation above is incorrect (see https://github.com/SigmaHQ/pySigma/blob/main/sigma/modifiers.py)
```

Comment:
The sigma spec does not actually define a comparator at all https://sigmahq.io/docs/basics/modifiers.html and calls everything a modifier. 

The question here is if we should retain a simpler API in the code and allow the modifier itself to support the OR operation or we should generalize the API to allow multiple outputs from the modifer to do the OR in this code.

I think it might be simpler to make base64 modifier support the OR natively and keep the old API